### ISSUE TITLE
createDirectory not working

### ISSUE THREAD

**Issue by lijialiang** [skpm/dialog#8]
```js
const [output] = dialog.showOpenDialog({
    defaultPath: path.resolve(document.path, '../'),
    properties: ['openDirectory', 'createDirectory']
})
```

Did not see the button for 'new folder'

**Comment by Ashung** [comment]
https://github.com/skpm/dialog/blob/master/lib/open-dialog.js#L51-L52

@mathieudutour 

```
dialog.canCreateDirectories = true
```

**Comment by mathieudutour** [comment]
Thanks, fixed in 0.2.5
